examples/csharp/source/wrapper.d

```diff
@@ -0,0 +1,26 @@
+module csharp.wrapper;
+
+import csharp.library;
+
+import autowrap.csharp.boilerplate;
+import autowrap.reflection;
+
+mixin(
+    wrapCSharp("csharp",
+        Modules(
+            Module("csharp.library")
+        )
+    )
+);
+
+version (GenerateCSharp) {
+void main() {
+    import std.stdio;
+    import autowrap.csharp.csharp : writeCSharpFile;
+    string csharpFile = writeCSharpFile!(Module("csharp.library"))("csharp", "csharp");
+    auto f = File("Wrapper.cs", "w");
+    f.writeln(csharpFile);
+    f.flush();
+    f.close();
```

Comment:
flush and close happens automatically on process exit, no?

Reply:
They should - this should only be added if necessary, which in all likelihood it isn't.

Reply:
Sure. But stating it outright doesn't hurt anything and it shows what's happening.

Reply:
It does - more code for equivalent functionality is always worse. The best code is no code at all. Anything that can be deleted should be.

We wouldn't need destructors if we preferred to manually write them everywhere.